VAMobile/src/components/FormWrapper/FormFields/VATextInput.test.tsx

```diff
@@ -1,14 +1,14 @@
 import 'react-native'
```

Comment:
remove line 1 import and the jest styled components import and react test instance import

Reply:
This can be removed, as Dylan mentioned

Reply:
Updated.